app/src/main/java/org/oppia/app/help/faq/FAQActivity.kt

```diff
@@ -6,10 +6,11 @@ import android.os.Bundle
 import android.view.Menu
 import org.oppia.app.R
 import org.oppia.app.activity.InjectableAppCompatActivity
+import org.oppia.app.help.faq.faqsingle.FAQSingleActivity
 import javax.inject.Inject
 
 /** The FAQ page activity for placement of different FAQs. */
-class FAQActivity : InjectableAppCompatActivity() {
+class FAQActivity : InjectableAppCompatActivity(), RouteToFAQSingleListener {
```

Comment:
Please rename FAQ to FAQs/FAQList if its focusing on more than one.

Reply:
Updated all files